src/sentry/models/commitfilechange.py

```diff
@@ -1,10 +1,33 @@
+from typing import Any, Iterable, Optional
+
 from django.db import models
 
-from sentry.db.models import BoundedPositiveIntegerField, FlexibleForeignKey, Model, sane_repr
+from sentry.db.models import (
+    BaseManager,
+    BoundedPositiveIntegerField,
+    FlexibleForeignKey,
+    Model,
+    sane_repr,
+)
 
 COMMIT_FILE_CHANGE_TYPES = frozenset(("A", "D", "M"))
 
 
+class CommitFileChangeManager(BaseManager):
+    def get_count_for_commits(
+        self, commits: Iterable[Any], organization_id: Optional[int] = None
```

Comment:
I'd probably recommend we just remove the `organization_id` param here, there's not much to gain from having it and we have the chance of someone misusing it.